### DISCUSSION TITLE
Annotation lab

### DISCUSSION THREAD

**Discussion by albin02t** [JohnSnowLabs/spark-nlp#13075] [General]
Is the annotation lab for NLP, video, audio in this same repository?

**Comment by maziyarpanahi** [comment]
Hi,

This repository hosts full access to codes/models/examples/docs/etc. for `Spark NLP | spark-nlp` (forever free & open-source under Apache 2.0 License for both personal and commercial uses).

At the same time, it also hosts documentaitons to other products (but only markdowns/docs without any source code) including ALAB.
